docs/guided-tour/concepts.md

```diff
@@ -1,27 +1,36 @@
 # Concepts
 
 Semaphore manages your build, test and deployment workflows with _pipelines_,
-_blocks_, and _promotions_. Workflows may contain multiple pipelines, for
-example one for tests and another for deployment. _Blocks_ define what to do
-at each step in the pipeline. _Promotions_ connect different pipelines.
-Semaphore schedules and runs the blocks on agents. Each agent manages the
-environment the blocks run in. All configuration is written in YAML files.
-The initial pipeline is always kept in `.semaphore/semaphore.yml`. Promoted
-pipelines have different filenames.
+_blocks_, and _promotions_:
+
+- Workflows may contain multiple pipelines, for example one to run tests and
+  another for deployment.
+- _Blocks_ define what to do at each step in the pipeline.
+- Blocks run in _agents_ that define the hardware and software environment.
+- _Promotions_ connect different pipelines.
+
+All configuration is specified in YAML files. The initial pipeline is always
+sourced from `.semaphore/semaphore.yml`. Additional pipelines triggered via
+promotions are defined in separate files.
 
 ![Semaphore 2.0 concepts diagram](https://storage.googleapis.com/semaphore-public-assets/public/images/semaphoreci2-concepts.png)
 
 ## Blocks & Tasks
 
-_Blocks_ are the heart of a pipeline. Each block has a _task_ that
-defines one or more _jobs_. Jobs define the commands to execute.
+Blocks are the building blocks of a pipeline. Each block has a _task_ defined by
```

Comment:
@markoa This first sentence looks weird to me wdyt?

Reply:
Why specifically?

Reply:
To me, it sounds weird that such a short sentence defines Blocks by being blocks.

Reply:
I see, I don't think I've realized the duplication until now. I'll think of something else async.

Reply:
I've thought about this one and probably the best idea that comes to mind is:
`Blocks are the building steps of a pipeline.`
The word step is already used above:
`_Blocks_ define what to do at each step in the pipeline.`
Anything else I can think of changes the sentence almost entirely.